projects/canopy/src/lib/progress-indicator/progress-indicator.component.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { LgProgressIndicatorComponent } from './progress-indicator.component';
+
+describe('LgProgressIndicatorComponent', () => {
+  let component: LgProgressIndicatorComponent;
+  let fixture: ComponentFixture<LgProgressIndicatorComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [ LgProgressIndicatorComponent ],
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(LgProgressIndicatorComponent);
+    component = fixture.componentInstance;
+  });
+
+  it('should create the component', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should have the correct default values', () => {
+    expect(component.max).toBe(0);
+    expect(component.value).toBe(0);
+    expect(component.showProgressBar).toBe(true);
+    expect(component.displayAs).toBe('step');
+    expect(component.stepsPrefix).toBe('Step');
+  });
+
+  it('should apply the correct CSS class', () => {
```

Comment:
Again, apologies for the super nit - as mentioned we usually put the CSS class test under the `it('should create the component'` test :)